pages/common/ani-cli.md

```diff
@@ -0,0 +1,28 @@
+# ani-cli
+
+> A cli to browse and watch anime (alone AND with friends).
```

Comment:
Why do you emphasize `alone AND with friends` here?

Reply:
```suggestion
> A cli to browse and watch anime.
```